orchestra/static/orchestra/common/components/checklist/js/directives.js

```diff
@@ -0,0 +1,243 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('orchestra.common.components.directives')
+    .directive('orchestraChecklist', ['$http', '$timeout', '$compile', 'orchestraService', orchestraChecklist]);
+
+  function orchestraChecklist($http, $timeout, $compile, orchestraService) {
+    return {
+      restrict: 'E',
+      scope: {
+        'data': '=',
+        'author': '=?',
+        'imagePrefix': '=?',
+      },
+      link: function(scope, el, attr) {
+        var sampleItem = {
+          'title': 'This is a sample checklist item. ' +
```

Comment:
From user research it appears we learned that folks might be tempted to use this with coarse granularity (e.g., "Review 1") rather than fine granularity (e.g., "Fix the spelling of `teh`").  Can the example here be biased toward fine granularity so folks can be primed to use it that way/